src/components/ha-climate-control.html

```diff
@@ -48,7 +51,7 @@
 
   static get properties() {
     return {
-      value: {
+      last_changed: {
```

Comment:
last_changed is changed internally - no need to add observer to it - you can just call the needed function.

`value` if passed to this component, so it needs to be declared and observed

Reply:
The observer doesn't function as it should. It generates a bug when reaching end of interval. 
Should I will just add method call at end of each increment method?